index.js

```diff
@@ -1,98 +1,96 @@
 const http = require("http");
 const { parse } = require("querystring");
 const {
-  predictAction,
-  predictInteraction,
-  logAction
+	predictAction,
+	predictInteraction,
+	logAction,
 } = require("./vacabot/helpers");
 const {
-  actionCheckVacationBalance,
-  actionDenyVacationRequest,
-  actionOpenCreateVacation,
-  actionSubmitVacationRequest,
-  actionUpdateManager
+	actionCheckVacationBalance,
```

Comment:
They weren't updated from 4 tab size to 2 tab size

Reply:
let me check